### PR TITLE
fix(windows): typo in font helper string

### PR DESCRIPTION
**Pull request by mcdurdin**

### CODE REVIEW

File: windows/src/desktop/kmshell/xml/strings.xml [hunk 1]
```diff
@@ -1193,7 +1193,7 @@ PRIVACY WARNING: Please note that the debug logfile records all keystrokes that
   <!-- Context: OSK Font Helper -->
   <!-- String Type: FormatString -->
   <!-- Introduced: 8.0.294.0 -->
-  <string name="S_OSK_FontHelper_PleaseWait" comment="OSK Font helper message">Please wait while searching for releted fonts for keyboard %0:s</string>
+  <string name="S_OSK_FontHelper_PleaseWait" comment="OSK Font helper message">Please wait while searching for related fonts for keyboard %0:s</string>
```
Comment (darcywong00) [inline windows/src/desktop/kmshell/xml/strings.xml:1196]: Ran a quick spell check, and found typos in comments that could be fixed (while in this file)

l. 442
permissible

l. 474
system tray

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.223-beta](https://keyman.com/downloads/releases/beta/14.0.223)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.9-alpha](https://keyman.com/downloads/releases/alpha/15.0.9)

**Pull request merged**